CONTRIBUTING.md

```diff
@@ -40,6 +42,21 @@ as well as data-persistence; with two additional submodules.
 - `features`, which includes the high-level test definitions for the Convene
   experience.
 
+### Architecture Overview
+
+_Last updated: Dec 2022_
+
+![](./docs/convene-architecture-overview.jpg)
+
+Original on Miro: https://miro.com/app/board/o9J_lLrbz1g=/
+
+### ERD
+
+![](./docs/erd.png)
+This Entity-relationship diagram is an overview of our models and how they are associated.
```

Comment:
Maybe everyone knows what an ERD is, but I like this explanation.

Reply:
I dig it! I've put it insdie of the alt-text for the image.